api/models/example.py

```diff
@@ -125,7 +126,8 @@ def create(
             all_model_annotation_data.update(input)
             all_model_annotation_data.update(output)  # this overwrites any user
             # inputs that are also model outputs
-            if not task.verify_annotation(all_model_annotation_data):
+            verified, _ = task.verify_annotation(all_model_annotation_data)
```

Comment:
Should we also log the message returned from verify annotation?

Reply:
Thanks for catching this. The error messages here don't get sent to the user interface, so I think this is ok for now.